packages/suite/src/views/suite/index.tsx

```diff
@@ -92,7 +93,12 @@ const Index: FunctionComponent<Props> = props => {
         );
     }
 
-    return <Layout showSuiteHeader>{props.children}</Layout>;
+    return (
+        <Layout showSuiteHeader>
+            {props.children}
+            <Modals />
```

Comment:
imaybe t would be better for consitency to move `Modals` to Layout component, where also `SuiteNotifications` is.

Reply:
moved